### ISSUE TITLE
Olympics Replay

### ISSUE THREAD

**Issue by YuJian-cn** [ytdl-org/youtube-dl#29851]
## Checklist

- [x] I'm reporting a new site support request
- [ ] I've verified that I'm running youtube-dl version **2021.06.06**
- [ ] I've checked that all provided URLs are alive and playable in a browser
- [ ] I've checked that none of provided URLs violate any copyrights
- [ ] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://olympics.com/tokyo-2020/en/replay/7dbb1441-691e-429f-a807-5109ba4c0a24/mens-individual-q-fs-s-fs-finals

## Description

东京奥运官方回看，很多比赛其它网站没有，特别是没有解说的视频。

**Comment by dirkf** [comment]
Google thinks OP said:
>Tokyo Olympics official looks back, many other websites in the game are not, especially the video that has not been explained.

**Comment by dmzimmerman** [comment]
Indeed, the Tokyo Olympics official replay website, https://olympics.com/tokyo-2020/en/replay/, is apparently (and unsurprisingly, since it didn't exist in June) unsupported by the 2021.06.06 version (which I have tried). I could check off all the items in the checklist above, as I've done all those things, but as this isn't my bug, I'm not sure about the etiquette of that.
